### PR TITLE
Issue #571. Change Browse Issues -> Latest Bugs

### PR DESCRIPTION
**Pull request by miketaylr**
Mostly removes a ton of code. I'm using the same markup patterns from issue-list.html, so the few issues we do show on the homepage will be "larger". But I think that's what the design from #571 shows (and it looks fine to me).

r? @magsout

### CODE REVIEW

**Comment by miketaylr** [comment]
`16/53 tests failed`. That's weird.

File: tests/functional/index.js [hunk 1]
```diff
@@ -25,35 +25,6 @@ define([
         .end();
     },
 
-    'my issues (when logged in)': function() {
-      return this.remote
-        .setFindTimeout(intern.config.wc.pageLoadTimeout)
-        .get(require.toUrl(url))
-        .findByCssSelector('.js-login-link').click()
```
Comment (miketaylr) [inline tests/functional/index.js]: Oops.

**Comment by miketaylr** [comment]
`2/53 tests failed` that's better! :whale2:

**Comment by magsout** [comment]
LGTM. Good job @miketaylr :+1:

**Comment by magsout** [comment]
Maybe last 10 bugs ?

**Comment by miketaylr** [comment]
> Maybe last 10 bugs ?

Yeah, agreed. Will change (and fix remaining failing tests).

**Comment by miketaylr** [comment]
OK, build # 814 passed. Good enough for me. :P

**Pull request merged**